flow/data_pipeline/query.py

```diff
@@ -0,0 +1,55 @@
+"""stores all the pre-defined query strings."""
+from enum import Enum
+
+# tags for different queries
+tags = {"energy": ["POWER_DEMAND_MODEL", "POWER_DEMAND_MODEL_DENOISED_ACCEL", "POWER_DEMAND_MODEL_DENOISED_ACCEL_VEL"],
+        "analysis": ["POWER_DEMAND_MODEL"]}
+
+
+class QueryStrings(Enum):
+    """An enumeration of all the pre-defined query strings."""
+
+    SAMPLE = "SELECT * FROM trajectory_table WHERE partition_name=\'{partition}\' LIMIT 15;"
```

Comment:
can we format these to look a little prettier using triple-quote docstrings? e.g.
```
POWER_DEMAND_MODEL = """
    SELECT
        id,
        time,
        speed,
        ...
    FROM trajectory_table
    WHERE partition_name={}
    ORDER BY id, time
""".format(partition)```